cashctrl_ledger/ledger.py

```diff
@@ -1,4 +1,4 @@
-"""Module to sync ledger system onto CashCtrl."""
+"""Module that implements the pyledger interface by connecting to the CashCtrl"""
```

Comment:
'the CashCtrl' is poor English. Also, we should not assume that 'the CashCtrl' is a known concept.

Suggestion:
```
"""Module that implements the pyledger interface by connecting to the CashCtrl API."""
```